lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -1202,11 +1202,23 @@ void InterpreterFunction::fwdElementCmpLTEInst(const ElementCmpLTEInst *I) {
 }
 
 void InterpreterFunction::fwdElementCmpEQInst(const ElementCmpEQInst *I) {
-  auto outW = getWeightHandle<int64_t>(I->getDest());
```

Comment:
I don't intend this to be the final solution for float tensor comparisons; I mainly wanted to get this PR out for discussion and feedback.

For the purposes of implementing this operator, I need to compare to a literal 0, so a fuzzy float comparison won't really be useful. How does everyone feel about a different instruction for float comparisons? Or an `IsZero` node, similar to the `IsNaN` node that was added recently?

Reply:
In that case, maybe you can put *WIP* in the title :)